plot/steer_aggregate_and_plot_observables.py

```diff
@@ -76,48 +77,178 @@
+# Note: all observables that are plotted are saved to hdf5, and are then written to tables
+# Note: plotting script may have warnings about missing histograms, this is usually due to some missing centrality bins for certain design points
+analysis_name = 'Analysis3'
+download_runinfo = True # if true, download all run_info.yaml files specified in DataManagement (runs.yaml)
+download_histograms = False # if true, download all histogram files from OSN
+list_paths_for_selected_design_points = False
+merge_histograms = True # if true, merge all histograms for each run into a single file
+aggregate_histograms = True # if true, aggregate all histograms for each design point into a single file
+plot_and_save_histograms = True # if true, plot and save all histograms
+write_tables = False # if true, write tables for input to Bayesian analysis
+plot_global_QA = False # if true, plot global QA
+
+# re-analysis parameters
+#-----------------------------------------------------------------
+download_final_state_hadrons = True # if False, final_state_hadrons.parquet files for local analysis
+analysis_final_state_hadrons = True  # if true, analyze final_state_hadrons.parquet files locally
+local_analysis_facility = "test_hiccup"
+
+# version number used to identify the version of the analysis. Histograms and observables are stored on OSN in facility/RunX/histograms/versionY
+# if version -1 is specified, histogram files for download will be downloaded from faclity/RunX/histograms without version specifier
+# if version -1 is specified, no upload of the results from reanalysis is possible
+analysis_version = -1 
+force_reanalysis = True # force to analyse and download again, even if histogram files are present
+delete_final_state_hadrons_after_analysis = True # delete final state hadron files after analysis is done
+
+# Processing options
+#-----------------------------------------------------------------
+force_download = False
+download_threads = 5
+n_cores = 20 # only used for merging with hadd
+ntasks = 80 # number of analysis tasks to be submitted to local_analysis_facility
+max_n_slurm_jobs = 80
+skip_run_binomial = True
+
+# Location where histogram files and final state hadron files are stored on OSN
+#-----------------------------------------------------------------
+facilities = ['bridges2', 'expanse']
+
+# Settings related to software and data locations
+#-----------------------------------------------------------------
+stat_xsede_2021_dir = '/software/users/fjonas/myJETSCAPE/STAT-XSEDE-2021'
+jetscape_analysis_dir = '/software/users/fjonas/myJETSCAPE/JETSCAPE-analysis'
+local_base_outputdir = '/rstorage/fjonas/jetscape/aggregation_data'
+analyis_container_path = '/rstorage/jetscape/containers/local/stat_local_gcc_v3.6.sif'
+
+# debug options
+#-----------------------------------------------------------------
+ranomize_run_order = True  # if true, runs will be shuffled into random order. this can be useful for benchmarking
```

Comment:
Move out of debug options or make a note that we want to run this way by default